flink-ml-lib/src/main/java/org/apache/flink/ml/clustering/kmeans/KMeans.java

```diff
@@ -253,59 +258,76 @@ public Tuple3<Integer, DenseVector, Long> map(Tuple2<Integer, DenseVector> value
             implements TwoInputStreamOperator<
                             DenseVector, DenseVector[], Tuple2<Integer, DenseVector>>,
                     IterationListener<Tuple2<Integer, DenseVector>> {
+
         private final DistanceMeasure distanceMeasure;
-        private ListState<DenseVector> points;
-        private ListState<DenseVector[]> centroids;
+
+        private ListState<DenseVector[]> centroidsState;
```

Comment:
nits: could we keep the original name `centroids` for simplicity? Same for points.